ngsa/src/main/resources/application.properties

```diff
@@ -37,13 +37,14 @@ spring.banner.location=classpath:static/startup-banner.txt
 server.shutdown=graceful
 spring.lifecycle.timeout-per-shutdown-phase=10s
 
-burst-service=ngsa-java
-burst-target=60
-burst-max=80
+burst-service=${BURST_SERVICE:ngsa-java}
+burst-target=${BURST_TARGET:60}
+burst-max=${BURST_MAX:80}
+burst-header=${BURST_HEADER:false}
 
 # Ngsa specific properties
 region=dev
 zone=dev
 port=8080
 request-log-level=info
```

Comment:
I know we set the log level in the commonutils class but not sure how this is different from the CLI arg we pass. @kforeverisback  thoughts?

Reply:
Looks like these different custom logs with metadata specific to each web requests as part of `RequestLogger.java`